### PR TITLE
Fixes #12, fixes #13 and fixes #14: Add native language clients generated from Swagger Codegen CLI

### LINKED ISSUE #12: Add Java client generated from Swagger Codegen CLI
It is possible to generate Java clients using Swagger Codegen CLI (swagger-codegen-cli), and we should add description on how to do it and sample configurations.

### LINKED ISSUE #13: Add Ruby client generated from Swagger Codegen CLI
It is possible to generate Ruby clients using Swagger Codegen CLI (swagger-codegen-cli), and we should add description on how to do it and sample configurations.

### LINKED ISSUE #14: Add Python client generated from Swagger Codegen CLI
It is possible to generate Python clients using Swagger Codegen CLI (swagger-codegen-cli), and we should add description on how to do it and sample configurations.

### PR DESCRIPTION
**Pull request by bczoma**
Provides code and tutorial how to generate the SEMP Client Library using swagger-codegen and also automated scripts to do the same.

### CODE REVIEW

**Comment by bczoma** [comment]
Travis build is now passing.

File: docs/_tutorials/generate-semp-client-lib.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+layout: tutorials
```
Comment (mdspielman) [inline docs/_tutorials/generate-semp-client-lib.md]: You have windows line endings. Let's clean this file up.

File: docs/_tutorials/java-client.md [hunk 2]
```diff
@@ -23,39 +23,21 @@ The examples below make a few assumptions for simplicity:
...
+Before sending any commands, you must initialize the client library. There is a close relationship between the tags used in the SEMP specification and the SEMP client library classes offered. Each tag, like `msgVpn`, is compiled into a class in the `com.solace.sempclient.samplelib.api` package. Because of this, you have the option to work with the full SEMP API or subsets as controlled by the tags in the SEMP specification. You should determine which tag best fits your use case and use this client API class. For this introduction, we will use the `msgVpn` tag and the `MsgVpnApi` class because configuring a Solace message router message-VPN is a very common task in a dev-ops workflow.
 
 Before you can send any commands to the Solace message router, you need an instance of the SEMP API. The following code shows you how to create such an instance and set the SEMP username and password. This will connect over HTTP.
 
 ```
-import io.swagger.client.ApiClient;
-import io.swagger.client.api.MsgVpnApi;
+import com.solace.sempclient.samplelib.ApiClient;
```
Comment (mdspielman) [inline docs/_tutorials/java-client.md]: Needs to be com.solace.labs....

This isn't a supported Solace product, so it shouldn't generate libraries in the com.solace space but instead use the labs.

File: java/codegen_config_java.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+   "modelPackage": "com.solace.sempclient.samplelib.model",
```
Comment (mdspielman) [inline java/codegen_config_java.json]: Make this `com.solace.labs.sempclient.samplelib.model`

**Comment by bczoma** [comment]
Addressed Mark's comments.

**Pull request merged**